src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side = 0;
+
+    public Square(int side) {
+        this.side = side;
+    }
```

Comment:
```suggestion
public class Square extends Figure {
    private int side;

    public Square(String color, int side) {
        super(color);
        this.side = side;
    }
```
change other classes accordingly